include/overlay_123.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GUARD_POKEHEARTGOLD_OVERLAY_123_H
+#define GUARD_POKEHEARTGOLD_OVERLAY_123_H
+
+#include "global.h"
+
+BOOL ov123_0225F4A8(void (*)(void));
```

Comment:
Ideally we typedef a function pointer here

Reply:
i mean, it's overlay123 which is the biggest questionmark in all of johto. leaving as-is